requirements.txt

```diff
@@ -2,11 +2,15 @@
 click==8.1.3
 coverage==7.2.3
 grablib==0.8
-pytest==6.2.5
+pytest==7.3.1
 pytest-aiohttp==1.0.4
 pytest-cov==4.0.0
 pytest-forked==1.6.0
 pytest-mock==3.10.0
 pytest-sugar==0.9.7
 pytest-timeout==2.1.0
 pytest-toolbox==0.4
+aiohttp~=3.8.4
+Pygments~=2.15.1
+devtools~=0.11.0
+watchfiles~=0.19.0
```

Comment:
I'm not sure why these are not included before, but always use `==`. Dependabot will keep them updated.

Reply:
Done, updated via force push.